src/App.tsx

```diff
@@ -1,61 +1,29 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
```

Comment:
The `todosFromServer` should be prepared in the `App` component and passed to child components. Avoid importing it directly in non-root components as per the checklist requirement: "Prepare data in one place (App component) and pass it to child components."